src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -320,44 +320,20 @@ private unsafe int CompareString(ReadOnlySpan<char> string1, ReadOnlySpan<char>
             }
         }
 
-        internal unsafe int IndexOfCore(string source, string target, int startIndex, int count, CompareOptions options, int* matchLengthPtr)
+        internal unsafe int IndexOfPlatform(string source, string target, int startIndex, int count, CompareOptions options, int* matchLengthPtr)
```

Comment:
I do not think we use `*Plaform` name for platform-specific part anywhere. The platform specific parts are typically called `*Core`.